src/McBopomofo.cpp

```diff
@@ -623,6 +624,23 @@ void McBopomofoEngine::handleCandidatesState(
   candidateList->setSelectionKey(selectionKeys_);
   candidateList->setPageSize(selectionKeys_.size());
 
+  fcitx::CandidateLayoutHint layoutHint;
```

Comment:
This duplicates the code in lines 414-426, perhaps extract a util method for this?

Reply:
Fixed. :)